rockcraft.yaml

```diff
@@ -37,27 +38,27 @@ parts:
       echo "auth required pam_wheel.so use_uid" >> /etc/pam.d/su
       rm -rf /var/cache/apt/*
     organize:
-      conf: opt/spark/conf
+      conf: etc/spark/conf
       jars: opt/spark/jars
       bin: opt/spark/bin
       sbin: opt/spark/sbin
       python: opt/spark/python
-      kubernetes/dockerfiles/spark/entrypoint.sh: opt/entrypoint.sh
-      kubernetes/dockerfiles/spark/decom.sh: opt/decom.sh
+      kubernetes/dockerfiles/spark/entrypoint.sh: opt/spark/entrypoint.sh
+      kubernetes/dockerfiles/spark/decom.sh: opt/spark/decom.sh
       examples: opt/spark/examples
       kubernetes/tests: opt/spark/tests
-      data: opt/spark/data
+      data: ${SPARK_USER_DATA}
     stage:
-      - opt/spark/conf
+      - etc/spark/conf
       - opt/spark/jars
       - opt/spark/bin
       - opt/spark/sbin
-      - opt/entrypoint.sh
-      - opt/decom.sh
+      - opt/spark/entrypoint.sh
+      - opt/spark/decom.sh
       - opt/spark/examples
       - opt/spark/tests
-      - opt/spark/data
       - opt/spark/python
+      - ${SPARK_USER_DATA}
```

Comment:
@deusebio  We seem to have a problem with this. I assume that we can't create (`stage`) this directory, as it's outside of the scope of the ROCK structure creation workflows. Thus the directory won't be created (see [test failure here](https://github.com/canonical/charmed-spark-rock/actions/runs/5201118383/jobs/9383081195#step:8:377) ) 

What other location would you suggest? Just `$SPARK_USER_DATA` directly (instead of `$SPARK_USER_DATA/data`)?